package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
```

Comment:
let's advance the version of this package

Reply:
✔️